tests/integration/reqs/test_account_info.py

```diff
@@ -1,15 +1,23 @@
-from unittest import TestCase
+from unittest import IsolatedAsyncioTestCase
```

Comment:
p sure this isn't available until 3.8, which is why i was playing with the `later` package that backports to 3.7.1. that would require us officially not supporting 3.7.0, but i think that's ok

Reply:
Oh you're right, lemme see if I can get `later` to work